index.html

```diff
@@ -30,87 +36,72 @@ <h6 class="a">LET’S CONNECT</h6>
           <li class="i"><a href="#"><img src="images/medium.png" alt=""></a></li>
         </ul>
       </section>
+<!-- WORKS -->
```

Comment:
- [ ] OPTIONAL: your cards are already centred as required, however, I would suggest you decrease the padding to make it more beautiful and well designed like the Figma design cards.
![image](https://user-images.githubusercontent.com/80612925/172896814-b84b142e-f61a-4886-8c47-551097e6afc8.png)

**2.Figma design**
![image](https://user-images.githubusercontent.com/80612925/172896864-faae9ef6-3a43-4253-b071-12f572d50532.png)